app/mailers/mail_utilities/smoke_test_email_check.rb

```diff
@@ -0,0 +1,29 @@
+module MailUtilities
+  SmokeTestEmailCheck = Struct.new(:email_address) do
+    def matches?
+      smoke_test_email_regex.match(email_address)
+    end
+
+    private
+
+    def smoke_test_email_regex
+      %r{
+        \A             # match from the start of the string
+        #{local_part}
+        \+             # google address alias see: https://support.google.com/mail/answer/12096
+        [0-9a-z\-]{36} # RFC 4122 uuid extension
+        @              # its an email!
```

Comment:
As @dan-palmer's editor, I take full responsibility.  Inconsistent application, however--you let @kyliemulholland get away with it yesterday :see_no_evil: